sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosQueryRequestOptions.java

```diff
@@ -489,6 +489,15 @@ CosmosQueryRequestOptions setPartitionKeyRangeIdInternal(String partitionKeyRang
         return this;
     }
 
+    PartitionKeyDefinition getPartitionKeyDefinition() {
```

Comment:
Would we need the same for change feed request options and possibly readMany and/or bulk?

Reply:
Fixed in next iteration.